4981_ChatRoom/clientwindow.cpp

```diff
@@ -60,14 +64,76 @@ ClientWindow::~ClientWindow()
     delete ui;
 }
 
-/* clients message send button event handler */
+/********************************************************
+ *  Function:   on_cltSendButton_clicked()
+ *
+ *  Programmer: Alex Zielinski
+ *
+ *  Created: Mar 11 2017
+ *
+ *  Modified:
+ *
+ *  Desc:
+ *      Event handler for when the send button is
+ *      clicked on the clients side, notifying
+ *      that the user wants to send a message to
+ *      the chat
+ *******************************************************/
 void ClientWindow::on_cltSendButton_clicked()
 {
     char message[BUFLEN];
     getUIMessage(message);
-    sendToServer(message, BUFLEN);
+    formatMessage(message,"127.0.0.1",this);
```

Comment:
For the moment, as I need to talk to Alex about getting and storing the local machine's actual IP. It's purely a place holder.